src/FplBot.ConsoleApps/FplPlayerCommandHandler.cs

```diff
@@ -43,7 +44,7 @@ public async Task<HandleResponse> Handle(SlackMessage message)
                 });
             }
 
-            return new HandleResponse("OK");
+            return new HandleResponse(matchingPlayers);
```

Comment:
lol. jeg er smart

Reply:
Denne her endringen var bare for å kunne enhetsteste CommandHandlern - ikkenoe gæli ellers :)

Reply:
var det name greia som fucka opp da? 🤔